src/components/postgresConnect.ts

```diff
@@ -772,6 +776,24 @@ class PostgresDB {
     }
     return reviewAssignmentIds
   }
+
+  public isFullyAssignedLevel1 = async (applicationId: number) => {
```

Comment:
is this one actually used ? (I wonder if there is a use case for it in the futute)

Reply:
Yeah, it's no longer used but I just left it here as I thought it might come up again.